### ISSUE TITLE
Update Hugo params per warning

### ISSUE THREAD

**Issue by batpigandme** [tidyverse/tidyverse.org#305]
From the latest `blogdown::serve_site()`:
```
Page's .Hugo is deprecated and will be removed in a future release. Use the global hugo function.
WARN 2019/05/06 09:34:07 Page's .URL is deprecated and will be removed in a future release. Use .Permalink or .RelPermalink. If what you want is the front matter URL value, use .Params.url.
```

**Comment by batpigandme** [comment]
- [ ] Remove `.Hugo`
- [ ] Remove `.URL`

**Comment by batpigandme** [comment]
Closed in #350
